kale/utils/save_xlsx.py

```diff
@@ -0,0 +1,85 @@
+"""
+Authors: Lawrence Schobs, lawrenceschobs@gmail.com
+
+Functions to save results to Excel files.
+
+"""
+
+from typing import Any, Dict
+
+import numpy as np
+import pandas as pd
+
+
+def generate_summary_df(
+    results_dictionary: dict, cols_to_save: list, sheet_name: str, save_location: str
+) -> pd.DataFrame:
+    """
+    Generates pandas dataframe with summary statistics.
+    Designed for use in Quantile Binning (/pykale/examples/landmark_uncertainty/main.py).
+    Args:
+        results_dictionary (dict): A dictionary containing results for each quantile bin and uncertainty method.
+            The keys are strings indicating the name of each uncertainty method.
+            The values are dictionaries containing results for each quantile bin.
+        cols_to_save (list): A list of 2-element lists, each containing a string indicating the key in the
+            results_dictionary and a string indicating the name to use for that column in the output dataframe.
+        sheet_name (str): The name of the sheet to create in the output Excel file.
+        save_location (str): The file path and name to use for the output Excel file.
+    Returns:
+        pd.DataFrame: A dataframe with statistics including mean error, std error of All and individual
+        targets. Also includes the Sucess detection rates (SDR).
```

Comment:
This is a generic function. Is SDR generic enough for this API?

Reply:
i think so. SDR can be used for any continuous error metric (regression)